### ISSUE TITLE
UI incorrect language

### ISSUE THREAD

**Issue by FastusBellrune** [olivierkes/manuskript#1220]
I am not an English native speaker, but I write in English and I prefer the UI language of my writing programs to be in English.
When I started the software for the first time it was in my native language, but when I went to change it to English, on General settings, the language was already set to English.
To change it I had to select my language, restart the software, select English, and restart again.
It's not a huge problem, but since I wasn't sure that I had to change it there, I had some problems finding out how to do it.

OS: Windows 11
Manuskript version: 0.15.0

**Comment by obw** [comment]
I don't know which is your native Language, but I know not all Languages are complete Translated, but you can help to fix that:

https://hosted.weblate.org/projects/manuskript/translations/

**Comment by FastusBellrune** [comment]
I'm Italian, and it shows 99%, but that's not the issue. I prefer the English interface.

What I'm saying is that when I first started it I found it in Italian, and I wanted it in English. When I went to change it to English in "General settings" I found the language to be already set to English, even though the actual language shown by the UI was Italian.

To actually change the UI to English, I had to change the Language to Italian, restart the software, then change it to English and restart again.

**Comment by obw** [comment]
Strange behaviour!

**Comment by FastusBellrune** [comment]
Yes, it is. I don't remember exactly where, but I found out how to do it in a comment here, so I think it's not just mine. There's no issue open for it though.

**Comment by TheJackiMonster** [comment]
I think the title is now more clear, isn't it?
